topics/metabolomics/tutorials/msPurity_training/tutorial.md

```diff
@@ -0,0 +1,1006 @@
+>     - **Input** : please select the first option, the default parameter : 
+>       - `Prepared database of MassBank, HMDB, LipidBlast and GNPS` : database prepared by msPurity authors.
+>       - *SQLite database of (LC)-MS/MS data* : it is the database you created during **msPurity.spectralMatching {% icon tool %} tool** processing.
+>       - *Locally configured SQLite, MySQL or PostgreSQL database* : ?????????????????????????????????????????????
+>     - **Filters** : a lot of filters for this database (more in details section)
+> 
+>
+> > ### {% icon details %} Details about filters
+> > 
+> > Each database has its filters which are explain just below. You don't need to touch to them for a normal matching. But you should need some of these filters to have more precise results :
+> > {: .text-justify}
+> >   - **ppm error of the precursor for spectra** : the `value for the ppm error` of the precursor for spectra.
+> >   {: .text-justify}
+> >   - **ppm error of the product for spectra** : the `value for the ppm error` of the product for spectra.
+> >   {: .text-justify}
+> >   - **Filter on relative abundance threshold?** : if you want to filter on this parameter, set the value for your `relative abundance threshold`.
+> >   {: .text-justify}
+> >   - **Filter on polarity?** : if you want to filter on this parameter, enter and `select the polarity` or polarities you want to have in your datas.
+> >   {: .text-justify}
+> >   - **Filter on precursor ion purity?** : if you want to filter on precursor ion purity, set the `threshold` value for your datas.
+> >   {: .text-justify}
+> >   - **Filter on XCMS groups ids?** : if you want to filter on this parameter, add the `grpid` of each peak separate by a coma.
+> >   {: .text-justify}
+> >   - **Filter on pids?** : if you want to filter on this parameter, add the `pid` of each peak separate by a coma.
+> >   {: .text-justify}
+> >   - **Filter on retention time range?** : if you want to filter on this parameter, complete the `minimum retention time range and the maximum retention time range` you want for your datas.
+> >   {: .text-justify}
+> >   - **Filter on accessions?** : if you want to filter on this parameter, enter the `accesion ID` you don't want in your datas, separate by a coma.
+> >   {: .text-justify}
+> >   - **Filter on sources?** : if you want to filter on this parameter, `select databases` you want to use and delete each one you don't want.
+> >   {: .text-justify}
+> >   - **Filter on instrument types?** : if you want to filter on this parameter, `select intrument type` you want to keep and delete each one you don't want.
+> >   {: .text-justify}
+> >   - **Filter on instrument name?** : if you want to filter on this parameter, enter the `name of the intrument` if you know it. 
+> >   {: .text-justify}
+> >   - **Filter on spectral type?** : if you want to fil
```

Comment:
Does the Galaxy wrapper work here ? Or should µI just precised that it is in development?